controllers/error-handlers.js

```diff
@@ -0,0 +1,16 @@
+const psqlErrorCodes = {
+    '22P02': {status: 400, msg: 'invalid_text_representation'}
+}
```

Comment:
I'd maybe change this to something slightly more user friendly like 'Invalid Input', but I'll leave it up to you to decide 🙂

Reply:
This is a actually the description psql provides for that error code, I agree that it's not intuitive. Maybe it's worth waiting a bit until I've collected all relevant psql errors and then group them into something more natural?